.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on: 
+  push:
+    tags:
+      - '!player/'
+      - 'v*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up node.js
+        uses: actions/setup-node@v3.3.0
+        with:
+          node-version-file: .nvmrc
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Test
+        run: npm test
+
+      - name: Build
+        run: gulp build-prod
+
+      - name: Publish
+        run: bash publish.sh
```

Comment:
I guess we don't need the `bash` right?

Reply:
We could get rid of that and add the `shell` type in a separate parameter as shown in https://docs.github.com/en/actions/learn-github-actions/essential-features-of-github-actions#adding-scripts-to-your-workflow:

```yaml
jobs:
  example-job:
    steps:
      - name: Run build script
        run: ./.github/scripts/build.sh
        shell: bash
```
Not sure this is a huge improvement to the initial code, but changed it in 6bc9b515.